src/golang/lib/models/shared/storage.go

```diff
@@ -0,0 +1,48 @@
+package shared
+
+import (
+	"database/sql/driver"
+
+	"github.com/aqueducthq/aqueduct/lib/collections/utils"
+)
+
+type StorageType string
```

Comment:
Where is this file used btw or is this from a merge conflict?

Reply:
Doesn't seem to currently be in use. Deleting.

Reply:
Oh wait actually `GCSConfig` from the storage.go is being used in integration_config.go. I believe nothing else in the file is currently being used.